docs/tables/aws_clb_access_log/index.md

```diff
@@ -179,7 +183,7 @@ partition "aws_clb_access_log" "my_logs_account" {
   source "aws_s3_bucket" {
     connection  = connection.aws.logging_account
     bucket      = "aws-clb-logs-bucket"
-    file_layout = "AWSLogs/(%{DATA:org_id}/)?123456789012/elasticloadbalancing/%{DATA:region}/%{YEAR:year}/%{MONTHNUM:month}/%{MONTHDAY:day}/%{DATA}.log"
+    file_layout = "AWSLogs/(%{DATA:org_id}/)?123456789012/elasticloadbalancing/%{DATA:region}/%{YEAR:year}/%{MONTHNUM:month}/%{MONTHDAY:day}/%{NUMBER:account_id}_elasticloadbalancing_%{DATA:region}_[^_]+_[^_]+_[^_]+_[^.]+.log"
```

Comment:
I don't understand this change, we have a capture group for `account_id` and `region` prior to the `year` one, you can't redeclare them again.